packages/expo-av/android/build.gradle

```diff
@@ -77,7 +77,7 @@ dependencies {
   // to `app/build.gradle`, in Expo apps `expoview` depends on `expolib_v1` dependency
   // overridding CustomHeadersOkHttpDataSourceFactory, so the code that uses unprefixed
   // extension won't ever be executed.
-  provided 'com.google.android.exoplayer:extension-okhttp:2.6.1'
+  provided 'com.google.android.exoplayer:extension-okhttp:2.9.2'
```

Comment:
I think the comment above could also be removed and `extension-okhttp` promoted to `api` (instead of `provided`)